display.js

```diff
@@ -0,0 +1,52 @@
+export const getCellElem = function(cell){
```

Comment:
there's a bunch of extra newlines between the functions in this class, clean them up. should just be one newline between each function